dsps/__init__.py

```diff
@@ -1,4 +1,3 @@
 # flake8: noqa
 """
 """
-from ._version import __version__
```

Comment:
Keep this line

Reply:
I think because of this line of code, I can no longer just run `py.test` from root command line of the package anymore. I need to first have done `pip install .` so that `dsps/_version.py` is created. Otherwise there is an import error when running `py.test`. Is this expected/intended?